### DISCUSSION TITLE
Issues with overlapping dropdownmenu and carousel

### DISCUSSION THREAD

**Discussion by Charlottechee1997** [twbs/bootstrap#31415] [General]
Hi, is anyone have suggest how to move dropdownmenu to in front of the carousel? Cause I try it many times using z-index, it doesn't working. Please let me know which place to i need to add on. Thank you.

![2](https://user-images.githubusercontent.com/64122592/89258958-dad54780-d65b-11ea-80ff-17f17765b52a.PNG)

**Comment by Deckluhm** [comment]
Can you provide a live URL or a reproduction case using [CodePen](https://codepen.io/pen), [JSFiddle](https://jsfiddle.net) or anything similar?

I will be easier to help you.
